index.bs

```diff
@@ -1,14 +1,160 @@
- Group: AOM
- Status: WGD
- Title: Burst data description for the Immersive Audio Model and Formats (IAMF)
- Editor: Jan Skoglund, Google, jks@google.com
- Repository: AOMediaCodec/iac
- Shortname: iamf-burst
- URL: https://aomediacodec.github.io/iamf-burst/
- Date: 2022-1-11
- Abstract: This AOM document specifies the method of conveying non-linear PCM bitstreams applying IEC 60958 encoded according to the Immersive Audio Model and Formats (IAMF) format.
- </pre>
- 
- # Section 1
- This is an empty spec
\ No newline at end of file
+Group: AOM
+Status: WGD
+Title: Burst data description for the Immersive Audio Model and Formats (IAMF)
+Editor: Jan Skoglund, Google, jks@google.com
+Repository: AOMediaCodec/iamf-hdmi
+Shortname: iamf-burst
+URL: https://aomediacodec.github.io/iamf-burst/
+Date: 2025-03-14
+Abstract: This AOM document specifies the method of conveying non-linear PCM bitstreams applying [[!IEC-60958]] encoded according to the Immersive Audio Model and Formats ([[!IAMF]]) format.
+</pre>
+
+
+<pre class='biblio'>
+{
+	
+	"IEC-60958": {
+		"title": "IEC 60958 — Digital audio interface — (All parts)",
+		"status": "2025. International Standard",
+		"publisher": "IEC",
+		"href": "https://webstore.iec.ch/en/publication/4048"
+	},
+	"IEC-61937-1": {
+		"title": "Digital audio – interface for non-linear PCM encoded audio bitstreams applying IEC 60958 – Part 1: General",
+		"status": "2021. International Standard",
+		"publisher": "IEC",
+		"href": "https://webstore.iec.ch/en/publication/62828"
+	},
+	"IEC-61937-2": {
+		"title": "Digital audio – interface for non-linear PCM encoded audio bitstreams applying IEC 60958 – Part 2: Burst-info",
+		"status": "2021. International Standard",
+		"publisher": "IEC",
+		"href": "https://webstore.iec.ch/en/publication/66291"
+	},
+	"IAMF": {
+		"title": "Immersive Audio Model and Formats",
+		"status": "Standard",
+		"publisher": "AOM",
+		"href": "https://aomediacodec.github.io/iamf/v1.1.0.html"
+	},
+	"RFC2119": {
+		"authors": "S. Bradner",
+		"title": "Key words for use in RFCs to Indicate Requirement Levels",
+		"status": "March 1997. Best Current Practive",
+		"publisher": "IETF",
+		"href": "https://tools.ietf.org/html/rfc2119"
+	}
+	
+	
+}
+</pre>
+
+
+
+
+
+
+
+# Terms and definitions # {#terms}
+For the purposes of this document, the following terms and definitions apply.
+
+## IAMF audio data frame ## {#IAMF-audio-data-frame-def}
+bit stream generated by an IAMF encoder containing audio data and metadata representing a block of linear PCM samples according to the IAMF specification.
+
+
+## frame length ## {#frame-length-def}
+number of linear PCM audio samples per IAMF audio data frame. The IAMF specification allows for multiple codecs, sampling rates and frame sizes. This burst specification constrains the use case to IAMF streams only employing codecs at a sampling rate of 48 kHz and an analysis data frames (update rate) of 20 ms, which corresponds to a frame rate of 960 samples.
```

Comment:
"an analysis data frames" -> "an analysis data **frame**"
"frame rate of 960 samples" -> frame **length** of 960 samples?